server/index.js

```diff
@@ -113,37 +119,16 @@ app.get('/:current', (req, res) => {
 
 app.post('/comment', async (req, res) => {
   try {
-    const lastComment = await db.lastComment();
-
-    const newComment = {
-      comment_id: lastComment[0].comment_id + 1,
-      user_id: req.body.user_id,
-      song_id: req.body.song_id,
-      content: req.body.content,
-      time_stamp: req.body.time_stamp,
-    };
-
-    const storedComment = await db.saveComment(newComment);
-
-    res.status(201).send({
-      success: true,
-      data: storedComment,
-    });
-  } catch (error) {
-    console.error(error);
-    res.status(400).json({
-      success: false,
-      msg: error,
-    });
-  }
-});
-
-app.post('/comment/:id', async (req, res) => {
-  try {
-    const { id } = req.params;
+    let commentId;
```

Comment:
On line 131, you could economize by calling this `comment_id`, which leads to 131 simplifying to `comment_id,\n`.